portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+
+public final class Comment {
+  private final String value;
+  private final long timeMillis;
+
+  public Comment(String comment, long time){
+    this.value = comment;
+    this.timeMillis = time;
+  }
+
+  public static Comment entityToComment(Entity comment) {
```

Comment:
Since you'll be calling this method with `Comment.` prefixing it, it might be a little cleaner to name it something like `fromEntity`.

Calling code will now look like `Comment.fromEntity(entity)`.

Feel free to ignore :)